libpod/boltdb_state_internal.go

```diff
@@ -221,22 +229,45 @@ func readOnlyValidateConfig(bucket *bolt.Bucket, toCheck dbConfigValidation) (bo
 	}
 
 	dbValue := string(keyBytes)
+	ourValue := toCheck.runtimeValue
+
+	// Tolerate symlinks when possible - most relevant for OStree systems
+	// and rootless containers, where we want to put containers in /home,
+	// which is symlinked to /var/home.
+	if toCheck.isPath {
+		if dbValue != "" {
+			// Ignore ENOENT on both, on a fresh system some paths
+			// may not exist this early in Libpod init.
+			dbVal, err := filepath.EvalSymlinks(dbValue)
+			if err != nil && !os.IsNotExist(err) {
```

Comment:
should use errors.Is() same in the other places
But I am not sure I follow the ENOENT explanation? If a path does not exists podman cannot function anyway. It is running podman right now in this moment so you cannot possible ever expect a sane behavior if our storage dir is returning ENOENT.

Reply:
This mostly applies to the volumes directory. That's not created until we actually start making volumes - so a lot of commands can be run and still no volumes directory.

Reply:
ok sounds fine then. I am still confused as of why the behaviour changed but I guess for now the prio is to have it fixed and this like a simple way to do it.